osu.Game/VolumeControl.cs

```diff
@@ -85,23 +72,37 @@ protected override bool OnWheelUp(InputState state)
         {
             appear();
 
-            VolumeGlobal.Value += 0.05f;
+            if (VolumeMeterSample.Contains(state.Mouse.Position))

```

Comment:
rather than using contains logic here, i'd rather you moved it inside VolumeControl (and pass through the bindable).